spec.bs

```diff
@@ -2832,15 +2839,21 @@ threshold when responding to [=query k-anonymity count=].
 </div>
 
 <div algorithm>
-  To <dfn>query ad k-anonymity count</dfn> given an [=interest group=] |ig| and a [=URL=] |ad|:
-    1. Let |keyString| be the [=string/concatenation=] of the following strings separated with U+000A LF:
+  To <dfn>query k-anonymity cache</dfn> given a |hashCode|:
+    1. If the [=user agent=]'s [=k-anonymity cache=] contains [=k-anonymity record=] |record| with the key |hashCode|:
+      1. If the difference between [=current wall time=] and |record|'s [=k-anonymity record/timestamp=] is more than 7 days then return false.
+      1. return |record|'s [=k-anonymity record/is k-anonymous=] field for the |record|.
+    1. Else return false

```

Comment:
```suggestion
    1. Otherwise, return false.

```

This is more consistent with the style of most specs (assuming the "Otherwise/Else" here should match the first "If" in the very first step of this algorithm.)

Reply:
nit: missing comma. Your call whether you want to add it.

Reply:
Added